content/_includes/site/footer.njk

```diff
@@ -1,10 +1,21 @@
 {% import 'contact.macros.njk' as contact %}
 {% import 'utility.macros.njk' as utility %}
 
+{% set add_cta = page.url | addCallToAction %}
+{% if add_cta %}
+    {% include './cta.njk' %}
+{% endif %}
+
 <div
   data-layout="footer">
   {% if page.url != '/contact/' %}
     <aside aria-label="contact">
+      {% if not add_cta %}
+        <div class="contact-form-intro">
+          <h3>Let's chat about your web project</h3>
+          Fill out the form, <a href="https://calendly.com/oddbird/discovery"> schedule a call</a>, <a href="mailto:birds@oddbird.net"> email</a>, <a href="https://twitter.com/oddbird"> tweet</a>, or find us on <a href="https://github.com/oddbird">GitHub</a>.
```

Comment:
Wrap this in a paragraph tag, which will also help the tight spacing: 
![Screenshot 2022-10-31 at 1 50 42 PM](https://user-images.githubusercontent.com/104161/199098068-335fd41c-3670-4d1f-894d-1e03486d8dd0.jpg)